apps/volk_option_helpers.h

```diff
@@ -23,12 +41,12 @@ typedef enum
 
 class option_t {
   public:
-  option_t(std::string longform, std::string shortform, std::string msg, void (*callback)());
-    option_t(std::string longform, std::string shortform, std::string msg, void (*callback)(int));
-    option_t(std::string longform, std::string shortform, std::string msg, void (*callback)(float));
-    option_t(std::string longform, std::string shortform, std::string msg, void (*callback)(bool));
-    option_t(std::string longform, std::string shortform, std::string msg, void (*callback)(std::string));
-  option_t(std::string longform, std::string shortform, std::string msg, std::string printval);
+  option_t(std::string t_longform, std::string t_shortform, std::string t_msg, void (*t_callback)());
+    option_t(std::string t_longform, std::string t_shortform, std::string t_msg, void (*t_callback)(int));
+    option_t(std::string t_longform, std::string t_shortform, std::string t_msg, void (*t_callback)(float));
+    option_t(std::string t_longform, std::string t_shortform, std::string t_msg, void (*t_callback)(bool));
+    option_t(std::string t_longform, std::string t_shortform, std::string t_msg, void (*t_callback)(std::string));
+  option_t(std::string t_longform, std::string t_shortform, std::string t_msg, std::string t_printval);
 
   std::string longform;
```

Comment:
Another option here that might be fewer changes is to move these to `d_` and leave the instantiation arguments alone. This would better fit into the "GR/Volk class variable naming" way of doing things ... but clearly either would work.

Reply:
I opted to `t_` because members are used in quite a few places and I tried to keep the changes to a minimum.

Reply:
works for me